deploy/deploy.go

```diff
@@ -827,7 +825,8 @@ func (i *IxiaTGSpec) Deploy(ctx context.Context) error {
 	if i.ConfigMap == nil {
 		path := filepath.Join(i.ManifestDir, "ixia-configmap.yaml")
 		if _, err := osStat(path); err != nil {
-			return fmt.Errorf("ixia configmap not found: %v", err)
+			log.Warnf("ixia controller deployed without configmap")
```

Comment:
why is this an option?

Reply:
Because we can't provide a config map due to non open sourced images (well I guess we technically can but this step would always fail unless the user has the correct secret setup)